### ISSUE TITLE
Question about disparity image.

### ISSUE THREAD

**Issue by polar99** [jzbontar/mc-cnn#43]
Hello.
In KITTI2012 data_stereo_flow\training\disp_noc\000000_10.png, I saw the maximum raw value is 17873. Is it the real disparity? Does it need any transformation?
Thanks a lot.

**Comment by saeid-h** [comment]
Yes. The image format is PNG 16bit. If you divide them by 256, you get the value that you are familiar with (i.e. PNG 8bit).
